ios/help/basic/keyboard-usage.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Using a Keyboard - Keyman for iPhone and iPad Help
+---
+
+Keyman keyboards are accessed by pressing the globe key whenever an iOS app presents the system keyboard. The follow screenshot shows the Keyman EuroLatin keyboard displayed with Apple's Calendar app.
+
+![](../ios_images/system-keyboard-cal.png)
+
+### Using Gestures on the Keyboard Keys
+
+Gestures provide ways to interact with the keyboard to produce different text output. Which gestures are available varies depending on the design of the individual Keyman keyboard. 
+
+**Note:** a dot or a small letter in the upper right corner of a key is a hint to indicate that the key supports flick, long press or multitap.
+
+The Gestures:
+
+* **Tap:** Tap on a character key to cause the associated character to be output to the app.
+
+* **Long press:** Press and hold a key, and after a moment a submenu will appear. Continue holding the key and slide to the submenu to select a key. Release, and the selected key from the submenu is output. A key submenu is shown in the following screenshot.
+
+![](../ios_images/key-submenu.png)
+
+* **Flick:** hold and slide in various directions on a key to reach alternate outputs. The key will animate to show you the expected output. A common flick is to slide down to reach numerals on the top row of the keyboard.
```

Comment:
The way this is worded suggests that it actually _is_ common... but at the moment, it'll be anything _but_.

Reply:
Thanks -- rephrasing that.